src/mudlet.pro

```diff
@@ -128,30 +128,33 @@ VERSION = 4.17.2
 # put something distinguishing into the MUDLET_VERSION_BUILD environment
 # variable (it should use '-' as the first character) to make identification of
 # the used version simpler
-# Note: the qmake BUILD variable is NOT a built-in one
-# BUILD = $$(MUDLET_VERSION_BUILD)
-isEmpty( BUILD ) {
-# Possible values are:
-# "-dev" for the development build
-# "-ptb" for the public test build
+# Note: the qmake APP_BUILD variable is NOT a built-in one
+APP_BUILD = $$(MUDLET_VERSION_BUILD)
+isEmpty( APP_BUILD ) {
+# Possible values that could be seen are:
+# "-dev" for a development build
+# "-ptb..." for a public test build
 # "" for the release build
 # A core dev team member setting things up for a release should comment out the
-# following line - as the app-build.txt file must not contain anything (other
-# than whitespace) for a RELEASE build:
-   # BUILD = "-dev-"$${GIT_SHA1}
-   BUILD = ""
+# following line - which forces APP_BUILD to take on a value IF IT WAS NOT SET IN THE
```

Comment:
Adding more and more text is making this setup more complicated, not less complicated, unfortunately. Let's go in the opposite direction.